plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/BridgeVifDriver.java

```diff
@@ -376,6 +376,7 @@ private void deleteVnetBr(String brName) {
             command.add("-v", vNetId);
             command.add("-p", pName);
             command.add("-b", brName);
+            command.add("-d", String.valueOf(deleteBr));
```

Comment:
should `deleteBr` be used to add the -d command?

Reply:
yes @rhtyd